documentation/source/user_section/tutorials/shape-analysis.rst

```diff
@@ -12,12 +12,13 @@ This section demonstrates how to compute shape metrics of the spinal cord using
 .. toctree::
    :maxdepth: 1
 
-   shape-metric-computation/before-starting
-   shape-metric-computation/csa-averaged
-   shape-metric-computation/csa-perlevel
-   shape-metric-computation/csa-pmj
-   shape-metric-computation/csa-perslice
-   shape-metric-computation/other-shape-metrics
-   shape-metric-computation/verifying-correctness
+   shape-analysis/before-starting
+   shape-analysis/csa-averaged
+   shape-analysis/csa-perlevel
+   shape-analysis/csa-pmj
+   shape-analysis/csa-perslice
```

Comment:
I suggest merging all four CSA entries into a single entry called "Cross Sectional Area (CSA)"

Reply:
done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4162/commits/bdc95b828658e5b0959e5b12d12e9095caedd10c and https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4162/commits/579da00b078526a0c0f6ac0b2eafdab53ec5ae59